src/App.js

```diff
@@ -120,6 +122,18 @@ const Guess = ({ guess, matches, word }) => {
   );
 };
 
+const welcome = ("Guess the five letter word");
```

Comment:
Let's put these in a single object:
```
const messages = {
  welcome: "Guess the five letter word",
  newMatched3: ["Are you clairvoyant!?", "Time to buy some lotto tickets"],
  ...
};
```
This is a general pattern for javascript messaging.

Reply:
easy enough.